### PR TITLE
simple eip-4844 interface for mcl

### PR DESCRIPTION
**Pull request by gnojus**
Tests passed for me locally.

### CODE REVIEW

File: mcl/kzg/rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+max_width = 110
```
Comment (sauliusgrigaitis) [inline mcl/kzg/rustfmt.toml:1]: Do we really need custom linter config? We prefer defaults mostly.

Comment (gnojus) [reply mcl/kzg/rustfmt.toml]: Good catch, this didn't need to be committed. Note that the whole source tree is not rustfmt'ed, I just wanted to format single working file and I'm not a fan of aggressive wrapping. Removed.

**Comment by sauliusgrigaitis** [comment]
Seems it's broken https://github.com/sifraitech/kzg/actions/runs/3446437047/jobs/5751336744 . Please fix.

**Comment by gnojus** [comment]
Sorry, opened #172

**Pull request merged**